README.md

```diff
@@ -27,38 +27,150 @@ That said, keep in mind the following:
   - [User Stories](https://docs.google.com/document/d/1wT8e8wBHMrSRHY4UF_60GCgyWGqvYye4THvaDARPySs/edit)
   - [Features](https://docs.google.com/document/d/1JNP6DgSK-c6KqveIhQk-n_HAw3hsZkL-okoleM43NgA/edit)
   - [Design Space](https://docs.google.com/document/d/1TpQlQYovCoX9FkpgsoxzdvZplghudHAiQOame30A-v8/edit)
-- [Frequently Asked Questions](FAQ.md)
+- [Frequently Asked Questions](docs/FAQ.md)
 
-## Usage
+## Setup
 
 Get the tool via
 
 ```sh
 $ go get -u github.com/golang/dep/cmd/dep
 ```
 
-Typical usage on a new repo might be
+To start managing dependencies using dep, run the following from your project root directory:
 
 ```sh
 $ dep init
-$ dep ensure -update
 ```
 
-To update a dependency to a new version, you might run
+This does the following:
+
+1. Look for [existing dependency management files](docs/FAQ.md#what-external-tools-are-supported) to convert
+1. Check if your dependencies use dep
+1. Identify your dependencies
+1. Back up your existing `vendor/` directory (if you have one) to
+`_vendor-TIMESTAMP/`
+1. Pick the highest compatible version for each dependency
+1. Generate [`Gopkg.toml`](Gopkg.toml.md) ("manifest") and `Gopkg.lock` files
+1. Install the dependencies in `vendor/`
+
+## Usage
+
+There is one main subcommand you will use: `dep ensure`. `ensure` first makes
+sure `Gopkg.lock` is consistent with your `import`s and `Gopkg.toml`, and then
+that `vendor/` is populated with exactly what's described in `Gopkg.lock`. `dep
```

Comment:
Sorry to add something at the last minute but someone was _just_ tripped up by this definition of `dep ensure` yesterday. https://github.com/golang/dep/issues/865#issuecomment-316838394

```
`ensure` first makes sure `Gopkg.lock` is consistent with your `import`s and `Gopkg.toml`. Then if any changes are detected,  populates `vendor/` with exactly what's described in `Gopkg.lock`.
```

We don't validate that the vendor directory matches the lock currently, and hopefully this will make that more clear! 😀